tests/wasm/runtime/abi.ts

```diff
@@ -348,7 +403,7 @@ export class Abi {
   }
 }
 
-function nextPowerOfTwo(value: number): number {
+function nextPowerOfTwoUint32(value: number): number {
```

Comment:
Maybe we should assert that value is not > max u32?

Reply:
This was added in a separate PR #106 https://github.com/gnosis/dex-subgraph/blob/86e603f08b8bfb231545b00f4c0332c07d16de3b/tests/runtime/abi.ts#L28-L31 based on your previous suggestion of creating an `allocate` helper function that ensures all allocations are done to the nearest power of 2.

Once this PR is rebased, this method should no longer exist.